llvm/llvm.spec.in

```diff
@@ -5,21 +5,20 @@ Summary: LLVM (An Optimizing Compiler Infrastructure)
 License: Apache-2.0 with LLVM exception
 Vendor: None (open source)
 Group: Development/Compilers
-URL: http://llvm..org/
-Source: http://llvm.org/releases/@PACKAGE_VERSION@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
+URL: https://llvm.org/
+Source: https://llvm.org/releases/@PACKAGE_VERSION@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
 BuildRoot: %{_tmppath}/%{name}-root
 Requires: /sbin/ldconfig
-BuildRequires: gcc >= 3.4
+BuildRequires: gcc >= 7.4
```

Comment:
Shouldn't this be also filled in by CMake using the @...@-variables? It wasn't updated since gcc 3.4 so I doubt anyone will remember to do so in the upcoming decade...

Reply:
Addressed